src/dashboard/src/components/transfer/views.py

```diff
@@ -156,6 +156,7 @@ def encoder(obj):
                 item['jobs'].append(newJob)
                 newJob['uuid'] = job.jobuuid
                 newJob['type'] = job.jobtype
+                newJob['link_id'] = job.microservicechainlink.pk
```

Comment:
A question outside of CR, but with these values in `transfer/views/py` and `ingest/views.py` could they be inherited from elsewhere instead of defining them twice?

Reply:
Yes, they're almost identical.

Reply:
During a big refactor that was never merged because of its size I tried to fix that:

https://github.com/artefactual/archivematica/pull/576/files#diff-254823927ae6fbe4584582f13c9b0c18R327

I learned the lesson and now I keep my work laser focused, e.g. see the second take of workflow translations here: https://github.com/artefactual/archivematica/issues/1101.